.bazelrc

```diff
@@ -0,0 +1,3 @@
+build:bzlmod --experimental_enable_bzlmod
+build:bzlmod --crosstool_top=@rules_cc.0.0.1.cc_configure.local_config_cc//:toolchain
```

Comment:
Please remove those flags, the toolchains are automatically registered, and we currently use the one embedded in bazel_tools instead of in rules_cc

Reply:
Done